seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -1596,7 +1596,7 @@
         "Here's an example of a radio button:",
         "<code>&#60;label&#62;&#60;input type=\"radio\" name=\"indoor-outdoor\"&#62; Indoor&#60;/label&#62;</code>",
         "<hr>",
-        "Add a pair of radio buttons to your form. One should have the option of <code>indoor</code> and the other should have the option of <code>outdoor</code>."
+        "Add a pair of radio buttons to your form. One should have the option of <code>indoor</code> and the other should have the option of <code>outdoor</code>. All two should share the <code>name</code> attribute of <code>indoor-outdoor</code>"
```

Comment:
Ah, sorry, I should have looked at that. For consistency, keeping the `share` part if fine, but please update the `All two` part. You are right that they do say `All three` in the next challenge, but in English we generally use `both` instead of `all` when we're talking about a fixed amount of two.

_Sidenote: I just checked out your GitHub profile and wow, you've really been doing a lot this year! That's impressive!_